content/news/XXXX-XX-XX-github-issues-migration.md

```diff
@@ -0,0 +1,86 @@
+(myself included), stay away from Launchpad. Especially for new contributors the
+mandatory Ubuntu One account was another reason for not to immediately report a
+bug when they encountered it.
+
+We've started transitioning away from Launchpad quite a while ago:  
+Back in 2013 we have transitioned from the Bazaar version control system to Git
+and moved our `main` branch to GitHub. At that point in time, GitHub issues where
+not yet suitable for managing our bugfix and release workflow, thus bug reports
+continued to be handled on Launchpad. This has progressively changed over the
+past couple of years, now GitHub has a "duplicate issue" detection feature, can
+organize bugs in projects and the labeling feature is flexible enough to fillful
+most of our needs.
+
+Going forward, Launchpad will only be used for publishing to our Ubuntu PPA. So
+no contributor outside of the core-team will have to bother with Launchpad
+again.
+
+Shortly before starting the migration, we made our Launchpad bug tracker
+read-only (some of you might have noticed that). The migration took about XX
+Hours. All of the existing messages were copied to GitHub unfortunately we were
+not able to migrate attachments and patches, but since we won't delete the
+launchpad bugtracker, you can still find those attachments there. For future
+reference: decided on the following mapping from Launchpad to Github issue
+labels:
+
+| Launchpad  | GitHub |
+| ------------- | ------------- |
+| Confirmed | `confirmed` |
+| Fix Committed | *close issue*  |
+| Fix Released | *close issue*  |
+| Incomplete | `incomplete` |
+| In Progress | *no label* |
+| Invalid | `invalid` |
+| New | *no label* |
+| Triaged | `confirmed` |
+| Won't Fix | `wontfix` |
+| Critical | `party stopper`,  `bug` |
+| High | `bug` |
+| Low | `bug` |
+| Medium | `bug` |
+| Undecided | *no label* |
+| Wishlist | `feature` |
+
+Since Mixxx is a project exclusively maintained by volunteers where each
+volunteer is free to work on whatever they choose anyways, we decided to drop
+the different priority levels assigned to issues.  
+
+I want to express a special thanks to our core team member
+[@Holzaus]({author}jan-holthuis) for writing a [custom import
+script](https://gist.github.com/Holzhaus/ed384b93465dcc516ae205090e4f179b) which
+made this migration feasible in the first place.
+
+This migration was also only possible because GitHub made a tremendous effort to
+help us. We noticed quite quickly that our import script was going to be
+struggling because of GitHub's secondary rate limiting. After
+[getting in contact with them via
+twitter](https://twitter.com/mixxxdj/status/1546544041038422019) they assembled
+a team of engineers and FOSS experts to help us evaluate possible solutions for
+the rate limitting problem and to avoid email spam from the imported issues.
+Thanks to them, we were able to avoid email spam and reduce the time the import
+took by two orders of magnitude.
```

Comment:
We may meld this paragraph a bit. Here is a first try: 
```suggestion
Due to the big number of bugs and the flood of related notifications I have noticed quickly that running the script will likely hit various rate limit or spam protection facility.  
After contacting GitHub team via
twitter](https://twitter.com/mixxxdj/status/1546544041038422019), they made a tremendous effort to
help us with a team of engineers and FOSS experts. 
Thanks to them, we were able to avoid email spam and reduce the time the import
took by two orders of magnitude.
```

Reply:
Sure, that works too.

Reply:
To be honest, after trying to work out some of the spelling mistakes and so forth, I'm not very happy how some parts sound. Can you point out what exactly you didn't like about the first version? 

What you do think about this:
```
Due to the big number of bugs and the flood of related notifications, I have
noticed quickly that the script will likely hit various rate limits and spam
protection facilities.  
After [getting in contact with them via
twitter](https://twitter.com/mixxxdj/status/1546544041038422019) they assembled
a team of engineers and FOSS experts to help us evaluate possible solutions.
Thanks to them, we were able to avoid email spam and reduce the time the import
took by two orders of magnitude.
```

Reply:
Perfect. Thank you. 

> Can you point out what exactly you didn't like about the first version?

It was IMHO just too long and detailed compared to the rest of the blog post. That's all.

Reply:
Oh, I just noticed my intend was lost in translation. Ich meinte "zusammenschmelzen"

Reply:
> Perfect. Thank you.

Great. I amended the commit.